src/sentry/rules/conditions/existing_high_priority_issue.py

```diff
@@ -0,0 +1,66 @@
+from collections.abc import Sequence
+from datetime import datetime
+
+from sentry import features
+from sentry.event_manager import HIGH_SEVERITY_THRESHOLD
+from sentry.eventstore.models import GroupEvent
+from sentry.models.activity import Activity
+from sentry.models.group import Group
+from sentry.receivers.rules import has_high_priority_issue_alerts
+from sentry.rules import EventState
+from sentry.rules.conditions.base import EventCondition
+from sentry.types.activity import ActivityType
+from sentry.types.condition_activity import ConditionActivity, ConditionActivityType
+from sentry.types.group import PriorityLevel
+
+
+class ExistingHighPriorityIssueCondition(EventCondition):
+    id = "sentry.rules.conditions.high_priority_issue.ExistingHighPriorityIssueCondition"
+    label = "Sentry marks an existing issue as high priority"
+
+    def is_new_high_severity(self, state: EventState, group: Group | None) -> bool:
```

Comment:
Is this meant to be `is_existing_high_severity`?

Reply:
Added the following comment: 

```
# This is legacy code that should not be used since "projects:issue-priority" is enabled.    
# TODO(snigdha): Remove this when the flag is removed.
```